resources/views/calendar.blade.php

```diff
@@ -0,0 +1,22 @@
+@extends('layout')
+
+@section('headerScripts')
```

Comment:
I think you could move moment, fullcalendar, jquery, AND the inlin jquery call all into the footer and make it work. hit me up if you have questions how